main.sql

```diff
@@ -1,2 +1,11 @@
--- wow! look at this epic project
-CREATE DATABASE musik;
+CREATE TABLE genres (
+    genre_name text,
+    genre_desc text,
+    CONSTRAINT genres_key PRIMARY KEY (genre_name)
+);
```

Comment:
```suggestion
CREATE DATABASE musik;

CREATE TABLE genres (
    genre_name TEXT CONSTRAINT genres_key PRIMARY KEY,
    genre_desc TEXT
);
```
1) Can't forget to actually create the database, or else we'll get a merge conflict :( (important!!)
2) Capitalizing datatypes makes it more sexy (not super important)
3) Line 2 and line 4 can be consolidated into one just to be a lil more succinct (also not super important, but nice to have)